front/app/typings.ts

```diff
@@ -54,7 +65,7 @@ export interface ITab {
   label: string;
   url: string;
   active?: boolean | ((pathname: string) => boolean);
-  feature?: TAppConfigurationSetting;
+  feature?: TAppConfigurationSettingWithEnabled;
```

Comment:
Had a type issue with using useFeatureFlag because we have some of these in homepageSettings now (needs to be changed). Luckily, we don't use the feature prop of ITab for a feature that doesn't have its enabled value in appConfig, so I narrowed the type, to make sure people don't add a tab with such a feature name (e.g. events_widget).